front/src/components/AdminPanel/Panel/InsertQualificationScores/InsertQualificationScores.js

```diff
@@ -0,0 +1,652 @@
+                `sum-${currentParticipantsScores[i]._id}`
+            ).value;
+            currentParticipantsScores[i].Xs = document.getElementById(
+                `countX-${currentParticipantsScores[i]._id}`
+            ).value;
+            currentParticipantsScores[i].tens = document.getElementById(
+                `count10-${currentParticipantsScores[i]._id}`
+            ).value;
+        }
+    };
+
+    let updateSumX10 = (id) => {
+        let sumNumbers = 0;
+        let sumX = 0;
+        let sum10 = 0;
+        let val;
+        for (var i = 1; i <= seriesType; i++) {
+            val = document.getElementById(`i${i}-${id}`).value;
+            if (val !== "") {
+                if (!/\D/.test(val)) {
+                    if (parseInt(val) === 10) {
+                        sum10 += 1;
+                    }
+                    sumNumbers += parseInt(val);
+                } else if (val === "X" || val === "x") {
+                    sumX += 1;
+                    sum10 += 1;
+                    sumNumbers += 10;
+                }
+            }
+        }
+        document.getElementById(`sum-${id}`).value = sumNumbers;
+        document.getElementById(`countX-${id}`).value = sumX;
+        document.getElementById(`count10-${id}`).value = sum10;
+        updateGetAllCurrentScores();
+    };
+
+    let blockAndCleanArrowsInputs = (id) => {
+        for (var i = 1; i <= seriesType; i++) {
+            var inputEl = document.getElementById(`i${i}-${id}`);
+            inputEl.disabled = true;
+            inputEl.style.borderColor = "lightgrey";
+            inputEl.style.color = "lightgrey";
+            inputEl.style.boxShadow = "none";
+            inputEl.value = "";
+        }
+    };
+
+    let unblockAllInputs = (id) => {
+        for (var i = 1; i <= seriesType; i++) {
+            var inputEl = document.getElementById(`i${i}-${id}`);
+            inputEl.disabled = false;
+            inputEl.style.borderColor = "#bbcbdd";
+            inputEl.style.color = "#2755b9da";
+            inputEl.style.boxShadow = "none";
+        }
+
+        var sumInput = document.getElementById("sum-" + id);
+        var countXInput = document.getElementById("countX-" + id);
+        var count10Input = document.getElementById("count10-" + id);
+        sumInput.disabled = false;
+        sumInput.style.borderColor = "#bbcbdd";
+        sumInput.style.color = "#2755b9da";
+        countXInput.disabled = false;
+        countXInput.style.borderColor = "#bbcbdd";
+        countXInput.style.color = "#2755b9da";
+        count10Input.disabled = false;
+        count10Input.style.borderColor = "#bbcbdd";
+        count10Input.style.color = "#2755b9da";
+
+        $(`[id*='-${id}']`).hover(
+            function () {
+                $(this).css({ "box-shadow": "0 0 5px #6392b1da" });
+            },
+
+            function () {
+                $(this).css({ "box-shadow": "none" });
+            }
+        );
+    };
```

Comment:
Extract repeating lines to separate functions.
The more code you put in css files, the better - maybe it would be easier to set some predefined class to elements you want to format?